survival_guide/1_concepts.adoc

```diff
@@ -92,8 +92,16 @@ The table below lists the Fusion microservices deployed by our Helm chart. Recog
 
 |`logstash` |HTTP |StatefulSet |`system` |Not required. Minimum of 1 but 2 pods are recommended for HA |Collects logs from the other microservices and either indexes into system_logs or ships them to an external service like Elastic
 
+|`*-ui` |Web |Deployment |`system` |Not required; only 1 pod should be sufficient for most clusters |Stores logs and prior Argo workflow runs.
```

Comment:
I think this should be `argo-ui` the `*` makes it seem like it applies to `auth-ui`, `pm-ui` etc

Reply:
Changed.